code/style.css

```diff
@@ -1,58 +1,249 @@
+/*HEADER, BODY AND CARDS SECTION*/
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: Girassol;
+  background-image: url(./castleroom.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+header {
+  color: whitesmoke;
+  text-align: center;
+  font-family: Girassol;
```

Comment:
Let's not forget about font-fallback : without it, if there would be any problems with accessing Girassol font, browser would display everything in nasty, basic font.